openthread/CMakeLists.txt

```
@@ -3,66 +3,114 @@
+              $<TARGET_PROPERTY:bintools,strip_flag_infile>$<TARGET_FILE:openthread-mtd>
+              $<TARGET_PROPERTY:bintools,strip_flag_outfile>"${OPENTHREAD_DST_DIR}/$<TARGET_FILE_NAME:openthread-mtd>"
+              $<TARGET_PROPERTY:bintools,strip_flag_final>
+      COMMAND $<TARGET_PROPERTY:bintools,strip_command>
+              $<TARGET_PROPERTY:bintools,strip_flag>
+              $<TARGET_PROPERTY:bintools,strip_flag_debug>
+              $<TARGET_PROPERTY:bintools,strip_flag_infile>$<TARGET_FILE:openthread-radio>
+              $<TARGET_PROPERTY:bintools,strip_flag_outfile>"${OPENTHREAD_DST_DIR}/$<TARGET_FILE_NAME:openthread-radio>"
+              $<TARGET_PROPERTY:bintools,strip_flag_final>
+      COMMAND $<TARGET_PROPERTY:bintools,strip_command>
+              $<TARGET_PROPERTY:bintools,strip_flag>
+              $<TARGET_PROPERTY:bintools,strip_flag_debug>
+              $<TARGET_PROPERTY:bintools,strip_flag_infile>$<TARGET_FILE:openthread-cli-ftd>
+              $<TARGET_PROPERTY:bintools,strip_flag_outfile>"${OPENTHREAD_DST_DIR}/$<TARGET_FILE_NAME:openthread-cli-ftd>"
+              $<TARGET_PROPERTY:bintools,strip_flag_final>
+      COMMAND $<TARGET_PROPERTY:bintools,strip_command>
+              $<TARGET_PROPERTY:bintools,strip_flag>
+              $<TARGET_PROPERTY:bintools,strip_flag_debug>
+              $<TARGET_PROPERTY:bintools,strip_flag_infile>$<TARGET_FILE:openthread-cli-mtd>
+              $<TARGET_PROPERTY:bintools,strip_flag_outfile>"${OPENTHREAD_DST_DIR}/$<TARGET_FILE_NAME:openthread-cli-mtd>"
+              $<TARGET_PROPERTY:bintools,strip_flag_final>
+      COMMAND $<TARGET_PROPERTY:bintools,strip_command>
+              $<TARGET_PROPERTY:bintools,strip_flag>
+              $<TARGET_PROPERTY:bintools,strip_flag_debug>
+              $<TARGET_PROPERTY:bintools,strip_flag_infile>$<TARGET_FILE:openthread-ncp-ftd>
+              $<TARGET_PROPERTY:bintools,strip_flag_outfile>"${OPENTHREAD_DST_DIR}/$<TARGET_FILE_NAME:openthread-ncp-ftd>"
+              $<TARGET_PROPERTY:bintools,strip_flag_final>
+      COMMAND $<TARGET_PROPERTY:bintools,strip_command>
+              $<TARGET_PROPERTY:bintools,strip_flag>
+              $<TARGET_PROPERTY:bintools,strip_flag_debug>
+              $<TARGET_PROPERTY:bintools,strip_flag_infile>$<TARGET_FILE:openthread-ncp-mtd>
+              $<TARGET_PROPERTY:bintools,strip_flag_outfile>"${OPENTHREAD_DST_DIR}/$<TARGET_FILE_NAME:openthread-ncp-mtd>"
+              $<TARGET_PROPERTY:bintools,strip_flag_final>
+      COMMAND $<TARGET_PROPERTY:bintools,strip_command>
+              $<TARGET_PROPERTY:bintools,strip_flag>
+              $<TARGET_PROPERTY:bintools,strip_flag_debug>
+              $<TARGET_PROPERTY:bintools,strip_flag_infile>$<TARGET_FILE:openthread-rcp>
+              $<TARGET_PROPERTY:bintools,strip_flag_outfile>"${OPENTHREAD_DST_DIR}/$<TARGET_FILE_NAME:openthread-rcp>"
+              $<TARGET_PROPERTY:bintools,strip_flag_final>
```

Please do this in a loop.

```suggestion
foreach(target openthread-ftd openthread-mtd openthread-radio openthread-cli-ftd openthread-cli-mtd openthread-ncp-ftd openthread-ncp-mtd openthread-rcp)
    list(APPEND INSTALL_COMMANDS
      COMMAND $<TARGET_PROPERTY:bintools,strip_command>
              $<TARGET_PROPERTY:bintools,strip_flag>
              $<TARGET_PROPERTY:bintools,strip_flag_debug>
              $<TARGET_PROPERTY:bintools,strip_flag_infile>$<TARGET_FILE:${target}>
             $<TARGET_PROPERTY:bintools,strip_flag_outfile>"${OPENTHREAD_DST_DIR}/$<TARGET_FILE_NAME:${target}>"
              $<TARGET_PROPERTY:bintools,strip_flag_final>
    )
endforeach()
```